README.md

```diff
@@ -1,48 +1,69 @@
-## Inlang is not ready for use yet. Stay tuned, or get <a href="https://tally.so/r/3q4O59">notified</a>.
+### Inlang is not ready for use yet. Stay tuned, or get <a href="https://tally.so/r/3q4O59">notified</a>.
+#
 
 
 <div>
     <p align="center">
-        <img width="300" src="https://raw.githubusercontent.com/inlang/inlang/main/assets/logo-white-background.svg"/>
+        <img width="40%" src="https://raw.githubusercontent.com/inlang/inlang/main/assets/logo-white-background.svg"/>
     </p>
-    <h2 align="center">
-        Open Source Localization Solution for Software
-    </h2>
-    <!--
-    <h3 align="center">
-        <a href="https://inlang.dev/docs/getting-started" target="_blank">Get Started</a> · <a href="https://inlang.dev/docs/getting-started" target="_blank">Documentation</a> · <a href="https://inlang.dev/blog/inlang-v0-2" target="_blank">Blog</a>
-    </h3>
-    -->
+
 </div>
 
-# Inlang
 
-Inlang's goal is to make localization (i18n) of software effortless through developer tools, standardization and automation.
-  
+<h2 align="center">
+       A git-based software localization system
+</h2> 
+
+Inlang is a localization system that acknowledges git repositories as the single source of truth, utilizes CI/CD pipelines of repositories for automation, and extends git to close collaboration gaps between developers and translators. 
+
+<figure width="50%">
+  <img src="https://raw.githubusercontent.com/inlang/inlang/rfc-001-architecture/rfcs/assets/001-git-based-architecture.png" alt="Git-based architecture">
+</figure>
 
 ## Apps
 
 > :bulb: The apps and features you see below are the features that are available right now. More is planned.
 
 
-### [editor](apps/dashboard)  
+### [Editor](apps/dashboard)  
 Let non-technical team members and translators manage translations for you.
 
 ![dashboard-example](https://user-images.githubusercontent.com/35429197/154271089-9acf02c3-7c6e-435c-9014-6ee21426ab4d.png)
 
+- Interface built on top of git
```

Comment:
I would remove those points from the editor and abstract them to the project. 

Those points are not editor specific, seem out of place in the document, but are valid points. Abstracting them in the context of the project would lead to: 

> Interface built on top of git 
(The only remark that I would not abstract to a project level)

"A git-based translation management editor that combines the collaboration of cloud-based solutions with the simplicity of a local-first solution."